heat-stack/package.json

```diff
@@ -28,7 +28,9 @@
     "test:e2e:run": "cross-env CI=true playwright test",
     "test:e2e:install": "npx playwright install --with-deps chromium",
     "typecheck": "tsc",
-    "validate": "run-p \"test -- --run\" lint typecheck test:e2e:run"
+    "validate": "run-p \"test -- --run\" lint typecheck test:e2e:run",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build -o heat-stack/public/storybook-static"
```

Comment:
This line also needs to change to `"build-storybook": "storybook build -o public/storybook-static"`